starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,43 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(arr1){
```

Comment:
A few problems with this function you wrote.
First issue:
You wrote 
```js
  if (longestWord = ''){
    longestWord = undefined;
  }
```
You NEED either 2 equal signs or three equal signs but you wrote one equal sign.
```js
  if (longestWord === ''){ // <------- You need 3 equals in the condition
    longestWord = undefined;
  }
```
---------
Next Issue, this if statement does not do anything in the code.... So if we remove it, then we get this, which still works:
```js
function findLongestWord(arr1){
  var longestWord = '';

    for (var i =0; i < arr1.length; i++){
      if (longestWord.length < arr1[i].length){
        longestWord = arr1[i];
      }
   }
  return longestWord;
}
```